setup.cfg

```
@@ -19,10 +19,12 @@ classifiers =
 packages = find:
 install_requires =
     inifix>=0.4.2<0.5.0
-    lic
+    licplot
     matplotlib>=3.2.1
     numpy>=1.18.5
     rich
+    scikit-image
+    scipy
```

same suggestion here
```suggestion
    scikit-image
    scipy
```